util/all_tests.json

```diff
@@ -12,6 +12,16 @@
     "skip_sde": true,
     "skip_valgrind": true
   },
+  {
+    "cmd": ["crypto/crypto_test"],
+    "env": ["OPENSSL_armcap=0x0"],
+    "target_arch": "arm"
+  },
+  {
+    "cmd": ["crypto/crypto_test"],
+    "env": ["OPENSSL_armcap=0x1"],
```

Comment:
There is also OPENSSL_armcap=0x7D that we're using with ACVP, it's equivalent to neon+crypto in the removed simulation. I see that the selections for x86 below doesn't cover all combinations we're testing for ACVP, but at least for Arm, I think 0x7D is important.
Note: testing OPENSSL_armcap=0 and 1 on Graviton2 EC2, the least significant bit which is for ARMV7_NEON doesn't make a difference, since, as per its name, it's for armv7.

Reply:
0x7D is covered by default -- the first test in the file. Assuming we are running on GV, but I think that's a safe assumption. Regarding 0x1, that means we can remove the test, right?

Reply:
It doesn't hurt to keep it. I did some tests locally and it seems to be slower (??) than when set to 0. Needs more investigation.